### PR TITLE
Rules advanced options

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Adds notes about advance options for security monitoring rules 

### Motivation
Jira request

### Preview
https://docs-staging.datadoghq.com/sarina/include-exclude-signals/security_monitoring/detection_rules/security_monitoring

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_monitoring/detection_rules/security_monitoring.md [hunk 1]
```diff
@@ -44,6 +44,10 @@ Add additional queries with the Add Query button.
 
 **Note**: The query applies to all Datadog events and ingested logs which do not require indexing.
 
+#### Advanced options
+
+Click the **Advanced** option to add queries that will **Only trigger a signal when** a value is met, or **Never trigger a signal** when a value is met. For example, if a user is triggering a signal, but their actions are benign and no longer want signals triggered from this user, create a logs query that excludes `@user.username: john.doe` under `Never Trigger A Signal`.
```
Comment (kayayarai) [inline content/en/security_monitoring/detection_rules/security_monitoring.md:49]:
```suggestion
Click the **Advanced** option to add queries that will **Only trigger a signal** when a value is met, or **Never trigger a signal** when a value is met. For example, if a user is triggering a signal, but their actions are benign and you no longer want signals triggered from this user, create a logs query that excludes `@user.username: john.doe` under **Never Trigger A Signal**.
```
I think? Not sure where the word "when" belongs in each of these, inside or outside the bold tags. And not sure about monospace tagging on a UI element.

File: content/en/security_monitoring/detection_rules/security_monitoring.md [hunk 2]
```diff
@@ -83,6 +87,10 @@ Select a value to detect, learning duration, and, optionally, define a signal gr
 
 For example, create a query for successful user authentication and set **detect new value** to `country` and group by to `user`. Set a learning duration of `7 days`. Once configured, logs coming in over the next 7 days are evaluated with the set values. If a log comes in with a new value after the learning duration, a signal is generated, and the new value is learned to prevent future signals with this value.
 
+#### Advanced options
+
+Click the **Advanced** option to add queries that will **Only trigger a signal when** a value is met, or **Never trigger a signal** when a value is met. For example, if a user is triggering a signal, but their actions are benign and no longer want signals triggered from this user, create a logs query that excludes `@user.username: john.doe` under `Never Trigger A Signal`.
+
```
Comment (kayayarai) [inline content/en/security_monitoring/detection_rules/security_monitoring.md:93]: Make same changes, if any, as above.

**Review by kayayarai — CHANGES_REQUESTED** [review]

File: content/en/security_monitoring/detection_rules/security_monitoring.md (hunk 1 above)
Comment (sarina-dd) [reply content/en/security_monitoring/detection_rules/security_monitoring.md:49]: Good catch on the UI element, definitely should've just been bold. Confirmed, it's **Never trigger a signal when:** in the UI, so I'll update.

File: content/en/security_monitoring/detection_rules/security_monitoring.md [hunk 3]
```diff
@@ -44,6 +44,10 @@ Add additional queries with the Add Query button.
 
 **Note**: The query applies to all Datadog events and ingested logs which do not require indexing.
 
+#### Advanced options
+
+Click the **Advanced** option to add queries that will **Only trigger a signal when:** a value is met, or **Never trigger a signal when:** a value is met. For example, if a user is triggering a signal, but their actions are benign and no longer want signals triggered from this user, create a logs query that excludes `@user.username: john.doe` under the **Never trigger a signal when:** option.
```
Comment (kayayarai) [inline content/en/security_monitoring/detection_rules/security_monitoring.md:49]:
```suggestion
Click the **Advanced** option to add queries that will **Only trigger a signal when:** a value is met, or **Never trigger a signal when:** a value is met. For example, if a user is triggering a signal, but their actions are benign and you no longer want signals triggered from this user, create a logs query that excludes `@user.username: john.doe` under the **Never trigger a signal when:** option.
```

File: content/en/security_monitoring/detection_rules/security_monitoring.md [hunk 4]
```diff
@@ -83,6 +87,10 @@ Select a value to detect, learning duration, and, optionally, define a signal gr
 
 For example, create a query for successful user authentication and set **detect new value** to `country` and group by to `user`. Set a learning duration of `7 days`. Once configured, logs coming in over the next 7 days are evaluated with the set values. If a log comes in with a new value after the learning duration, a signal is generated, and the new value is learned to prevent future signals with this value.
 
+#### Advanced options
+
+Click the **Advanced** option to add queries that will **Only trigger a signal when** a value is met, or **Never trigger a signal** when a value is met. For example, if a user is triggering a signal, but their actions are benign and no longer want signals triggered from this user, create a logs query that excludes `@user.username: john.doe` under `Never Trigger A Signal`.
```
Comment (kayayarai) [inline content/en/security_monitoring/detection_rules/security_monitoring.md:92]:
```suggestion
Click the **Advanced** option to add queries that will **Only trigger a signal when** a value is met, or **Never trigger a signal** when a value is met. For example, if a user is triggering a signal, but their actions are benign and you no longer want signals triggered from this user, create a logs query that excludes `@user.username: john.doe` under `Never Trigger A Signal`.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
One more tiny thing in each

**Comment by sarina-dd** [comment]
@kayayarai Updated 😄

**Review by kayayarai — APPROVED** [review]

**Pull request merged**